### PR TITLE
DEVDOCS-4645: [update] Orders V3, clarify async payment processing

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-4645]

## What changed?
Added additional details regarding the asynchronous behavior of payment processing

## Anything else?

[DEVDOCS-4645]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4645?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/orders.v3.yml [hunk 1]
```diff
@@ -20,8 +20,7 @@ paths:
     post:
       summary: Capture
       description: |-
-        Capture the payment for an order updating the `payment_status` to `capture pending`.
-
+        Capture the payment for an order updating the `payment_status` to `capture pending`. The capture process is successful when there are no validation issues. However, the payment request happens as a scheduled job performing asynchronously.
```
Comment (bc-andreadao) [inline reference/orders.v3.yml:23]:
```suggestion
        Capture the payment for an order updating the `payment_status` to `capture pending`. The capture process is successful when there are no validation issues. However, the payment request is scheduled and asynchronous.
```

File: reference/orders.v3.yml [hunk 2]
```diff
@@ -67,7 +66,7 @@ paths:
     post:
       summary: Void
       description: |-
-        Void the payment for an order updating the `payment_status` to `void pending`.
+        Void the payment for an order updating the `payment_status` to `void pending`. The void process is successful when there are no validation issues. However, the void payment request happens as a scheduled job performing asynchronously.
```
Comment (bc-andreadao) [inline reference/orders.v3.yml:69]:
```suggestion
        Void the payment for an order updating the `payment_status` to `void pending`. The void process is successful when there are no validation issues. However, the void payment request is scheduled and asynchronous.
```

File: reference/orders.v3.yml [hunk 3]
```diff
@@ -274,7 +273,7 @@ paths:
     post:
       summary: Create a Refund
       description: |-
-        Creates a refund.
+        Creates a refund. The refund process is successful when there are no validation issues. However, the refund payment request happens as a scheduled job performing asynchronously.
```
Comment (bc-andreadao) [inline reference/orders.v3.yml:276]:
```suggestion
        Creates a refund. The refund process is successful when there are no validation issues. However, the refund payment request is scheduled and asynchronous.
```

**Review by bc-andreadao — APPROVED** [review]
Left an (optional) copyedit suggestion

File: reference/orders.v3.yml [hunk 4]
```diff
@@ -67,7 +66,7 @@ paths:
     post:
       summary: Void
       description: |-
-        Void the payment for an order updating the `payment_status` to `void pending`.
+        Void the payment for an order. When there are no payment method validation issues, the void process is successful, the `payment_status` updates to `void pending`, and the void payment request is scheduled. The void payment request occurs asynchronously.
```
Comment (slsriehl) [inline reference/orders.v3.yml:69]:
```suggestion
        Void the payment for an order. When there are no payment method validation issues, the void process is successful, the `payment_status` updates to `void pending`, and the void payment request is scheduled. The payment request itself occurs asynchronously.
```

File: reference/orders.v3.yml [hunk 5]
```diff
@@ -274,7 +273,7 @@ paths:
     post:
       summary: Create a Refund
       description: |-
-        Creates a refund.
+        Creates a refund. When there are no payment method validation issues, the refund process is successful, and the refund payment request is scheduled. The refund payment request occurs asynchronously.
```
Comment (slsriehl) [inline reference/orders.v3.yml:276]:
```suggestion
        Creates a refund. When there are no payment method validation issues, the refund process is successful and the refund payment request is scheduled. The payment request itself occurs asynchronously.
```
- list of two doesn't have a separating comma

**Review by slsriehl — APPROVED** [review]
👏

**Pull request merged**